enhancements/sig-architecture/15-resourcebasedscheduling/metadata.yaml

```diff
@@ -8,5 +8,7 @@ approvers:
   - "@qiujian16"
   - "@mdelder"
 creation-date: 2021-08-06
-last-updated: 2021-08-25
-status: provisional
+last-updated: 2021-09-17
+status: implemented
+see-also:
+  - "/enhancements/sig-architecture/15-resourcebasedscheduling/README.md"
```

Comment:
point to the original placement proposal https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/6-placements

Reply:
fixed